### PR TITLE
replace cas with oidc

### PR DESCRIPTION
**Pull request by srkirkland**
Will require new settings/secret file for dev and deployment

### CODE REVIEW

File: src/PeopleLookup.Mvc/appsettings.json [hunk 1]
```diff
@@ -5,10 +5,13 @@
     }
   },
   "Authentication": {
+    "Authority": "https://ssodev.ucdavis.edu/cas/oidc",
     "IamKey": "External",
     "CasBaseUrl": "https://ssodev.ucdavis.edu/cas/",
     "CasBaseUrlProd": "https://cas.ucdavis.edu/cas/",
```
Comment (jSylvestre) [inline src/PeopleLookup.Mvc/appsettings.json]: Remove these if they are not used anymore?

File: src/PeopleLookup.Mvc/Models/AuthSettings.cs [hunk 2]
```diff
@@ -7,8 +7,11 @@ namespace PeopleLookup.Mvc.Models
 {
     public class AuthSettings
     {
+        public string Authority { get; set; }
         public string IamKey { get; set; }
         public string CasBaseUrl { get; set; }
```
Comment (jSylvestre) [inline src/PeopleLookup.Mvc/Models/AuthSettings.cs:12]: Remove?

**Review by jSylvestre — APPROVED** [review]

File: src/PeopleLookup.Mvc/appsettings.json (hunk 1 above)
Comment (srkirkland) [reply src/PeopleLookup.Mvc/appsettings.json]: yea i think we should remove the CAS* ones

File: src/PeopleLookup.Mvc/Models/AuthSettings.cs (hunk 2 above)
Comment (srkirkland) [reply src/PeopleLookup.Mvc/Models/AuthSettings.cs:12]: authority is important and needed

Comment (jSylvestre) [reply src/PeopleLookup.Mvc/Models/AuthSettings.cs:12]: @srkirkland Sorry, clicked on the wrong line. Meant the CasBaseUrl  here.

**Pull request merged**